_source/_posts/2022-02-28-spring-native-jhipster.adoc

```diff
@@ -0,0 +1,451 @@
+---
+#keywords: spring native
+layout: blog_post
+title: "Spring Native: start your JHipster + Spring Boot app in just 1s️!"
+#title: "Spring Native: starting your Java apps in seconds!"
+#title: "Introducing JHipster Native: serverless full-stack Java for all ⚡️"
```

Comment:
I like this one and the one on #line 5, and so does the analyzer....  But I'd lose the !